File: yt/yt/flow/examples/java/word_count/wordcount/src/main/java/tech/ytsaurus/flow/examples/wordcount/model/Word.java
package tech.ytsaurus.flow.examples.wordcount.model;

import javax.persistence.Entity;

import tech.ytsaurus.flow.row.FlowMessage;

// [BEGIN stream_context]
@Entity
@FlowMessage(streamIds = {"words"})
public class Word {
    private String word;

    public Word() {
    }

    public Word(String word) {
        this.word = word;
    }

    public String getWord() {
        return word;
    }

    public void setWord(String word) {
        this.word = word;
    }
}
// [END stream_context]

File: yt/yt/flow/examples/kotlin/word_count/wordcount/src/main/kotlin/tech/ytsaurus/flow/examples/wordcount/model/Word.kt
package tech.ytsaurus.flow.examples.wordcount.model

import tech.ytsaurus.flow.row.FlowMessage
import javax.persistence.Entity

// [BEGIN stream_context]
@Entity
@FlowMessage(streamIds = ["words"])
class Word {
    var word: String? = null

    constructor()

    constructor(word: String?) {
        this.word = word
    }
}
// [END stream_context]
